### ISSUE TITLE
Issues in cart modal with large number of assets

### ISSUE THREAD

**Issue by a-karthik** [Adobe-Marketing-Cloud/asset-share-commons#467]
**Describe the bug**
When deleting asset in cart modal (which has more than 100 assets) using inline asset removal button then the cart view does not get refreshed as there is Error 414: "Request-URI Too Long" since there is a GET request for fetching cart partial. Please provide a solution for that.

**Environment**
* AEM Version: 6.5.2
* Asset Share Commons Version: 1.8.0
* Author, Publish or both: Publish

**To Reproduce**
1. Go to Asset share home page.
2. Add more than 100 assets to cart (exact threshold unknown; might happen for 80, 90+ assets as well).
3. Click on Remove inline with the asset.
4. Remove disappears but asset remains in cart and the count badge gets reduced.

**Expected behavior**
Cart should behave the same way when it has a small number of assets.

**Comment by a-karthik** [comment]
@davidjgonzalez Can I please have an update on this? as this needs to be fixed ASAP. An ETA for the fix would be helpful.
Thanks,
Karthik

**Comment by godanny86** [comment]
@a-karthik I think this function: https://github.com/Adobe-Marketing-Cloud/asset-share-commons/blob/master/ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/cart/clientlibs/site/js/cart.js#L59 Needs to be updated to POST request in order to avoid passing each of the Asset paths as query parameters. 

It might* be as simple as updating the following but I haven't tested...

```diff
      function updateCartModal() {
-            $.get(CART_URL, serialize()).then(function (htmlResponse) {
+            $.post(CART_URL, serialize()).then(function (htmlResponse) {
                ns.Elements.update(htmlResponse, WHEN_CART_UPDATED);
            });
        }
```

If you can test and file a Pull Request it will help expedite getting this bug fixed!

**Comment by a-karthik** [comment]
@godanny86 Thank you for the inputs Daniel. I have tried this but the problem with this is that the modal gets refreshed i.e. loaded from top again. It does not stay at the removed asset position which happens with GET.

**Comment by a-karthik** [comment]
@godanny86 Thank you for the inputs. It is working as expected. I will do the same.

**Comment by a-karthik** [comment]
@godanny86 Will the fix be released in upcoming version 1.8.1? An ETA for the release would be helpful.

**Comment by godanny86** [comment]
hi @a-karthik , we will look to include this in 1.8.2 and target a release of January 9th/10th...
